cinnamon-take-screenshot@duracell80/files/cinnamon-take-screenshot@duracell80/cinnamon-take-screenshot.sh

```diff
@@ -67,17 +67,23 @@ fi
 
 # IF GNOME SCREEN SHOT AVAILABLE - TODO SUPPORT OTHER NON MINT ENVIRONMENTS
 # OFFER THE AREA SELECTION TOOL, SAVE DIRECTLY TO DIRECTORY AND OPEN IN PIX
-if [[ $(compgen -c | grep -iw 'gnome-screenshot' | head -n1 | wc -l) == "0" ]]; then
+if [[ $(command -v gnome-screenshot &> /dev/null; echo $?) -ne 0 ]]; then
     zenity --error --icon-name=security-high-symbolic --text="${LAN00}";
 else
     # US Date format: 2024-01-21 16-27-09 = +%Y-%m-%d %H:%M
     # ELSE USE UNIVERSAL SECONDS FOR FILE NAME
     TS=$(date +%s)
 
-   gnome-screenshot --area --file="${DIR_TGT}/screenshot-auto_${TS}.png"
-    if [[ $(compgen -c | grep -iw 'pix' | head -n1 | wc -l) == "0" ]]; then
-        exit
+    FILENAME="${DIR_TGT}/screenshot-auto_${TS}.png"
+
+    gnome-screenshot --area --file="$FILENAME"
+
+    if [[ ! -f "$FILENAME" ]]; then
+		notify-send -i camera-photo-symbolic -u low "The screenshot was not created, the operation was probably cancelled!"
```

Comment:
@duracell80 would you bother handling the localization here? And also you can update the text if you want to. You can send a PR to the branch `cinnamon-take-screenshot@duracell80-fixes` on [my repo](https://github.com/anaximeno/cinnamon-spices-actions/tree/cinnamon-take-screenshot%40duracell80-fixes) and it will reflect here.

Reply:
I'd actually noticed this and started working on implementing it. There's a folder with the strings in text files and that should be handled better. We can implement it independent of your changes though.

Reply:
So, should this be merged, and after that you send the update with the improvements on the translation, or better to wait for your update? (Say what you think to us also @duracell80)